### DISCUSSION TITLE
Starting with version 13 pictures for (Album)Artist will not be displayed in browse view anymore

### DISCUSSION THREAD

**Discussion by kp-org** [jcorporation/myMPD#1171] [General]
Hi,

I'm curious that nobody else discovered this, but starting with 13.0.0 until 13.0.4 the artist arts located under pics/Artist will no longer be shown in the browse view. Rolling back to 12.1.1 and all is fine. With loglevel debug the following appears in the logs (V13):

> Nov 14 11:00:54 heyra mympd: Handle API request "MYMPD_API_DATABASE_TAG_LIST"
> Nov 14 11:00:54 heyra mympd: MYMPD API request (6)(881473003) MYMPD_API_DATABASE_TAG_LIST: {"jsonrpc":"2.0","id":881473003,"method":"MYMPD_API_DATABASE_TAG_LIST","params":{"offset":0,"limit":500,"searchstr":"","tag":"AlbumArtist","sortdesc":false}}
> Nov 14 11:00:54 heyra mympd: Push response to queue for connection 6: {"jsonrpc":"2.0","id":881473003,"result":{"method":"MYMPD_API_DATABASE_TAG_LIST","data":[{"value":"AC/DC"},{"value":"Accept"},{"value":"Ace Frehley"},{"value":"Aerosmith"},
> ... snip ...
> {"value":"Yesterday And Today"},{"value":"Yngwie Johann Malmsteen"},{"value":"Yngwie Malmsteen"},{"value":"Yngwie Malmsteen's Rising Force"},{"value":"Zed Yago"},{"value":"ZZ Top"},{"value":"Кипелов"}],"totalEntities":279,"returnedEntities":279,"offset":0,"searchstr":"","tag":"AlbumArtist","pics":false}}
> Nov 14 11:00:54 heyra mympd: Entering mpd idle mode
> Nov 14 11:00:54 heyra mympd: Got API response for id "6"
> Nov 14 11:00:54 heyra mympd: Sending response to conn_id 6 (length: 6929): {"jsonrpc":"2.0","id":881473003,"result":{"method":"MYMPD_API_DATABASE_TAG_LIST","data":[{"value":"AC/DC"},{"value":"Accept"},{"value":"Ace Frehley"},{"value":"Aerosmith"},
> ... snip ...
> {"value":"Yesterday And Today"},{"value":"Yngwie Johann Malmsteen"},{"value":"Yngwie Malmsteen"},{"value":"Yngwie Malmsteen's Rising Force"},{"value":"Zed Yago"},{"value":"ZZ Top"},{"value":"Кипелов"}],"totalEntities":279,"returnedEntities":279,"offset":0,"searchstr":"","tag":"AlbumArtist","pics":false}}
> Nov 14 11:00:54 heyra mympd: Sending 6929 bytes to 6
> Nov 14 11:00:54 heyra mympd: Leaving mpd idle mode

That's all, no pictures will be severed. Maybe the issue is '"pics":false' in the third last line. At least there is no HTTP request.

Now the log of 12.1.1 looks different after 'Sending', even that it seems the json string is cut:

> ts"},{"value":"Velvet Viper"},{"value":"Venom"},{"value":"Vicious Rumors"},{"value":"Victor Smolski"},{"value":"Victory"},{"value":"Vinnie Moore"},{"value":"Vi
> Nov 14 09:00:08 heyra mympd: Sending 8674 bytes to 1073
> Nov 14 09:00:08 heyra mympd: HTTP request (1073): GET /tagart?tag=Artist&value=AC%2FDC
> Nov 14 09:00:08 heyra mympd: Handle tagart for "Artist": "AC_DC"
> Nov 14 09:00:08 heyra mympd: Absolut media_file: /var/lib/mympd/pics/Artist/AC_DC
> Nov 14 09:00:08 heyra mympd: Searching image file for basename "/var/lib/mympd/pics/Artist/AC_DC"
> Nov 14 09:00:08 heyra mympd: Serving file /var/lib/mympd/pics/Artist/AC_DC.png (image/png)
> ... and so on serving more files

Here the whole summary (totalEntities etc.) are not logged, I assume it's a buffer issue, the length is different also. However, this works.

Unfortunately I didn't find anything in the Changelog nor a documentation change. Any help appreciated. OS is Gentoo.

**Comment by jcorporation** [comment]
#1172
